### PR TITLE
Reset Digital Bid Adapter: add new bid adapter

### PR DESCRIPTION
**Pull request by evanmsmrtb**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: 'resetdigital',
  params: {
    pubId: '123',
    forceBid: true
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/resetdigitalBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  if (origins && origins.length > 0) {
+    return origins[origins.length - 1]
+  }
+}
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [ 'banner', 'video' ],
+  isBidRequestValid: function(bid) {
+    return (bid.params.pubId !== null || bid.params.zoneId !== null);
```
Comment (mmoschovas) [inline modules/resetdigitalBidAdapter.js:21]: This logic is specifically looking for null values, but not defining either of these params would result in undefined which passes this check and allows the ad units bid request. Can this be updated to check for all empty values?

File: modules/resetdigitalBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  },
+  buildRequests: function(validBidRequests, bidderRequest) {
+    let stack = (bidderRequest.refererInfo &&
+      bidderRequest.refererInfo.stack ? bidderRequest.refererInfo
```
Comment (mmoschovas) [inline modules/resetdigitalBidAdapter.js:25]: I think the value if the condition proves true was meant to be bidderRequest.refererInfo.stack, no?

**Review by mmoschovas — CHANGES_REQUESTED** [review]

**Comment by mmoschovas** [comment]
@evanmsmrtb Also will need docs PR for this bidder

**Review by mmoschovas — APPROVED** [review]
Thanks for making the changes. LGTM

**Review by patmmccann — CHANGES_REQUESTED** [review]
You must return advertiserDomains

File: modules/resetdigitalBidAdapter.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
+
+import * as utils from '../src/utils.js';
+import { config } from '../src/config.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+const BIDDER_CODE = 'resetdigital';
+
+function getDomain () {
```
Comment (patmmccann) [inline modules/resetdigitalBidAdapter.js:7]: Why aren't you using built in functions for this

**Comment by mmoschovas** [comment]
@evanmsmrtb can you please resolve lint errors on this?

**Comment by mmoschovas** [comment]
@evanmsmrtb also Patrick's comment regarding advertiserDomains needs to be addressed as well before we can move forward with this

**Comment by bretg** [comment]
docs PR https://github.com/prebid/prebid.github.io/pull/2967

**Review by patmmccann — APPROVED** [review]

**Pull request merged**